aneeshrelan/node-multi-tenant-example
# feat(multi-tenancy): add support to include tenant context in queue publisher

To be merged after https://github.com/aneeshrelan/node-multi-tenant-example/pull/3
- added support to append tenant context to all messages in the publish helper
- ack/nack on the message would be handled in `consumerTenantContextHandler.ts` (This can be altered based on whether the raw message or extracted message would be passed on to the consumer)
- Modify redis client usage for improved `devX` .


Test

- yarn pm2 start
- yarn pm2 logs
- Go to http://localhost:15672/
- publish `{"tenantId":"tenant1","message" : "testMessage"}` to `assessmentWorkers`
- publish `{"tenantId":"tenant2","message" : "testMessage"}` to `assessmentWorkers`
- Observe logs for assessmentWorker and scoringWorker
